scripts/get_results.py

```diff
@@ -219,23 +222,26 @@ def get_unique_params(parsed_results):
 
 
 def print_results(checklist):
-    path = setup_save("check")
-    outfile = f"{path}/checklist.md"
+    paths = setup_save("check")
+    paths.append(f"{args.PRIVATE_REPORTS_OUT}") # add the private reports folder to the list
     checklist = [r.tolist() for r in checklist]
 
     writer = MarkdownTableWriter(
         headers=["Model", "Test", "Train (few-shot)", "# samples", "Prompt", "# Ref", "Seed", "Strategy", "Done?"],
         value_matrix=checklist,
     )
-    writer.dump(outfile)
-
-    with open(f"{path}/checklist.csv", "w") as f:
-        for c in checklist:
-            index = len(c) - 1
-            c[index] = c[index].replace(":white_check_mark:", "Yes")
-            c[index] = c[index].replace(":heavy_multiplication_x:", "No")
-            f.write(f"{','.join(c)}\n")
-
+    for path in paths:
```

Comment:
Writing to both target directories seems easiest to have the checklist accessible and synced in both places.